lab-nicole/model/playlist.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const playlistSchema = Schema ({
+  title: { type: String, required: true },
+  genre: { type: String, required: true },
+  desc: { type: String, required: true },
+  userID: { type: Schema.Types.ObjectId, required: true },
```

Comment:
Careful with the capital D, usually is is just Id, and the discrepancy could create issues